psalm.baseline.xml

```diff
@@ -1,235 +1,2 @@
-    </UndefinedClass>
-    <UndefinedDocblockClass occurrences="1">
-      <code>Locator</code>
-    </UndefinedDocblockClass>
-  </file>
-  <file src="src/Faker/Provider/Base.php">
-    <UndefinedDocblockClass occurrences="1">
-      <code>Closure</code>
-    </UndefinedDocblockClass>
-  </file>
-  <file src="src/Faker/Provider/File.php">
-    <FalsableReturnStatement occurrences="1">
-      <code>false</code>
-    </FalsableReturnStatement>
-  </file>
-  <file src="src/Faker/Provider/PhoneNumber.php">
-    <InvalidReturnStatement occurrences="1">
-      <code>$imei</code>
-    </InvalidReturnStatement>
-    <InvalidReturnType occurrences="1">
-      <code>int</code>
-    </InvalidReturnType>
-  </file>
-  <file src="src/Faker/Provider/ar_SA/Address.php">
-    <UndefinedPropertyFetch occurrences="1">
-      <code>static::$cityPrefix</code>
-    </UndefinedPropertyFetch>
-  </file>
-  <file src="src/Faker/Provider/cs_CZ/Person.php">
-    <InvalidReturnStatement occurrences="1">
-      <code>$birthNumber</code>
-    </InvalidReturnStatement>
-    <InvalidReturnType occurrences="1">
-      <code>czech</code>
-    </InvalidReturnType>
-    <NonStaticSelfCall occurrences="2">
-      <code>static::birthNumber(static::GENDER_FEMALE)</code>
-      <code>static::birthNumber(static::GENDER_MALE)</code>
-    </NonStaticSelfCall>
-    <UndefinedDocblockClass occurrences="1">
-      <code>czech</code>
-    </UndefinedDocblockClass>
-  </file>
-  <file src="src/Faker/Provider/en_SG/Person.php">
-    <InvalidArrayOffset occurrences="1">
-      <code>$checksumArr[$checksum % 11]</code>
-    </InvalidArrayOffset>
-  </file>
-  <file src="src/Faker/Provider/is_IS/Address.php">
-    <InvalidPropertyAssignmentValue occurrences="1"/>
-    <UndefinedDocblockClass occurrences="2">
-      <code>Icelandic</code>
-    </UndefinedDocblockClass>
-  </file>
-  <file src="src/Faker/Provider/is_IS/Person.php">
-    <InvalidArgument occurrences="1">
-      <code>static::$middleName</code>
-    </InvalidArgument>
-    <InvalidArrayOffset occurrences="1">
-      <code>$ref[$i]</code>
-    </InvalidArrayOffset>
-    <InvalidPropertyAssignmentValue occurrences="3"/>
-  </file>
-  <file src="src/Faker/Provider/ja_JP/Text.php">
-    <UndefinedMethod occurrences="1">
-      <code>static::split($text)</code>
-    </UndefinedMethod>
-  </file>
-  <file src="src/Faker/Provider/pl_PL/Company.php">
-    <InvalidReturnStatement occurrences="1">
-      <code>implode('', $result)</code>
-    </InvalidReturnStatement>
-    <InvalidReturnType occurrences="1">
-      <code>14</code>
-    </InvalidReturnType>
-  </file>
-  <file src="src/Faker/Provider/pl_PL/Person.php">
-    <UndefinedDocblockClass occurrences="1">
-      <code>DateTime</code>
-    </UndefinedDocblockClass>
-  </file>
-  <file src="src/Faker/Provider/sl_SI/Person.php">
-    <NonStaticSelfCall occurrences="2">
-      <code>static::lastName()</code>
-      <code>static::lastName()</code>
-    </NonStaticSelfCall>
-  </file>
```

Comment:
I think these should stay as well.

Reply:
Found a way to correctly run psalm now, so reverted to main + update